### PR TITLE
update Accept header

### PR DESCRIPTION
**Pull request by edDimensi**
Pr by this https://github.com/knspriggs/go-twitch/issues/32

### CODE REVIEW

**Comment by edDimensi** [comment]
Hmm, sorry. I don't know why test not working. I learn golang 1 week, i thing rename constant don't break tests.

File: session.go [hunk 1]
```diff
@@ -60,7 +60,7 @@ func (session *Session) request(method string, url string, q interface{}, r inte
 	if requestError != nil {
 		return requestError
 	}
-	request.Header.Add("Accept", APIV3Header)
+	request.Header.Add("Accept", session.VersionHeader)
```
Comment (knspriggs) [inline session.go]: Should leave this as a constant. ie. `APIV5Header`.

File: constants.go [hunk 2]
```diff
@@ -3,8 +3,8 @@ package twitch
 import "net/url"
 
 const (
-	// APIV3Header - default v3 api header
-	APIV3Header = "application/vnd.twitchtv.v3+json"
+	// APIV5Header - default v3 api header
```
Comment (knspriggs) [inline constants.go]: Should add V5 header without removing the V3 header.

File: session.go (hunk 1 above)
Comment (edDimensi) [inline session.go]: why? How change ApiHeader then?

File: constants.go (hunk 2 above)
Comment (edDimensi) [inline constants.go]: V3 not work.
https://blog.twitch.tv/action-required-twitch-api-version-update-f3a21e6c3410

**Comment by edDimensi** [comment]
Return APIV3Header

File: session.go (hunk 1 above)
Comment (edDimensi) [inline session.go]:
```
	twitchSession, err := twitch.NewSession(
		twitch.NewSessionInput{
			ClientID:      "id",
			VersionHeader: twitch.APIV5Header,
		})
```
If not use VersionHeader from session, why i can change VersionHeader?

**Comment by knspriggs** [comment]
@edDimensi please hold on merging this until we can get a passing PR build :)

**Comment by edDimensi** [comment]
:) Sorry, i not understand what you say. English is my weak spot.

**Comment by knspriggs** [comment]
No worries. I'm just going to fix the tests before merging the change. I should able to fix it in a few hours.

**Comment by edDimensi** [comment]
Oh, okay. I will work with my fork your project. Then you merge it, change on you project.

**Comment by edDimensi** [comment]
Hi?

**Pull request merged**